src/test/java/uk/gov/ons/census/action/schedule/ActionRuleProcessorTest.java

```diff
@@ -99,92 +117,63 @@ public void testExecuteAllCasesField() {
     when(actionInstructionBuilder.buildFieldActionInstruction(any(Case.class), eq(actionRule)))
         .thenReturn(new uk.gov.ons.census.action.model.dto.instruction.field.ActionInstruction());
 
-    // when
     ActionRuleProcessor actionRuleProcessor =
         new ActionRuleProcessor(
-            actionRuleRepo, caseRepository, actionInstructionBuilder, null, rabbitFieldTemplate);
+            actionRuleRepo,
+            actionInstructionBuilder,
+            printFileDtoBuilder,
+            null,
```

Comment:
looks like ur missing the printCaseSelectedBuilder argument here

Reply:
fixed